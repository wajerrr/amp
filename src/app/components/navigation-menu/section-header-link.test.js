import React from 'react';
import renderer from 'react-test-renderer';
import SectionHeaderLink from './section-header-link';

describe('HeaderLink', () => {
  const headerLink = {
    headline: 'headerLink',
    id: 'header-link',
    url: {
      canonical: 'www.headerLink.com',
    },
  };
  it('should match snapshot for a normal headerLink', () => {
    const tree = renderer
      .create(<SectionHeaderLink link={headerLink} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
