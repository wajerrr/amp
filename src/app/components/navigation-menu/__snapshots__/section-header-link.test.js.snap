// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLink should match snapshot for a normal headerLink 1`] = `
<a
  className="css-1h5t0lq-StyledLink e2cf1b50"
  href="www.headerLink.com"
>
  headerLink
</a>
`;
